packages/bloom/lib/commands/bloom/INFO.ts

```diff
@@ -1,11 +1,63 @@
-import { RedisArgument, Command } from '@redis/client/dist/lib/RESP/types';
+import { RedisArgument, Command, UnwrapReply, NullReply, NumberReply, TuplesToMapReply, Resp2Reply, SimpleStringReply } from '@redis/client/dist/lib/RESP/types';
+
+export type BfInfoReplyMap = TuplesToMapReply<[
+  [SimpleStringReply<'Capacity'>, NumberReply],
+  [SimpleStringReply<'Size'>, NumberReply],
+  [SimpleStringReply<'Number of filters'>, NumberReply],
+  [SimpleStringReply<'Number of items inserted'>, NumberReply],
+  [SimpleStringReply<'Expansion rate'>, NullReply | NumberReply] 
+]>;
+
+export interface BfInfoReply {
+  capacity: NumberReply;
+  size: NumberReply;
+  numberOfFilters: NumberReply;
+  numberOfInsertedItems: NumberReply;
+  expansionRate: NullReply | NumberReply;
+}
 
 export default {
   FIRST_KEY_INDEX: 1,
   IS_READ_ONLY: true,
   transformArguments(key: RedisArgument) {
     return ['BF.INFO', key];
   },
-  // TODO
-  transformReply: undefined as unknown as () => any
+  transformReply: {
```

Comment:
https://github.com/redis/node-redis/blob/v5/docs/v4-to-v5.md?plain=1#L238
https://github.com/redis/node-redis/discussions/2506

I'm not sure if we should keep the keys as is for the sake of consistency or transform them for the sake of "friendly/javascripty"..

Reply:
it says there "to avoid unneccessary transformations" (and confusion but I honestly don't think that is the case).

In this case a transformation is neccessary anyways, so might as well make it friendly.  I also distinguish between "info" objects that are defined data structures, and generic maps that can contain any set of keys/values (as I'll discuss on the next one)